packages/backend-defaults/src/CreateBackend.test.ts

```diff
@@ -86,65 +85,34 @@ describe('createBackend', () => {
     ).toThrow('The core.pluginMetadata service cannot be overridden');
   });
 
-  it('should throw if an unsupported InternalSharedEnvironment version is passed in', () => {
-    expect(() =>
-      createBackend({
-        env: {} as any,
-      }),
-    ).toThrow(
-      "Shared environment version 'undefined' is invalid or not supported",
-    );
-    expect(() =>
-      createBackend({
-        env: { version: {} } as any,
-      }),
-    ).toThrow(
-      "Shared environment version '[object Object]' is invalid or not supported",
-    );
-    expect(() =>
-      createBackend({
-        env: { version: 'v2' } as any,
-      }),
-    ).toThrow("Shared environment version 'v2' is invalid or not supported");
-  });
-
   it('should prioritize services correctly', async () => {
```

Comment:
Unsure if this test is still relevant/logicaly needed here

Reply:
Yep I think we can remove it completely at this point